app/controllers/sentences_controller.php

```diff
@@ -773,6 +776,35 @@ public function search()
             $tags = implode(',', $tagsArray);
         }
 
+        // filter by list
+        $searchableLists = $this->SentencesList->getSearchableLists();
+        if (!empty($list)) {
+            $isSearchable = $this->SentencesList->isSearchableList($list);
+            if ($isSearchable) {
+                $sphinx['filter'][] = array('lists_id', $list);
+                $found = false;
+                foreach ($searchableLists as $rec) {
+                    if ($list == $rec['SentencesList']['id']) {
+                        $found = true;
+                        break;
+                    }
+                }
+                if (!$found) {
+                    $searchableLists[] = $isSearchable;
+                }
+            } else {
+                $ignored[] = format(
+                    /* @translators: This string will be preceded by
+                       “Warning: the following criteria have been
+                       ignored:” */
+                    __("“belongs to list number {listId}”, because this list does ".
```

Comment:
That's what I thought, but it was good to see the actual display with some sample lines. Now that I've seen it, I would recommend this wording:

"belongs to list number {listId}" (because list {listId} does not exist or cannot be searched)

By "cannot be searched", I assume you mean that it's a private list, right? In that case, you could alternatively say "or is a private list".